builtin/rebase--helper.c

```diff
@@ -38,10 +60,6 @@ int cmd_rebase__helper(int argc, const char **argv, const char *prefix)
 				ABORT),
 		OPT_CMDMODE(0, "make-script", &command,
 			N_("make rebase script"), MAKE_SCRIPT),
-		OPT_CMDMODE(0, "shorten-ids", &command,
-			N_("shorten commit ids in the todo list"), SHORTEN_OIDS),
-		OPT_CMDMODE(0, "expand-ids", &command,
-			N_("expand commit ids in the todo list"), EXPAND_OIDS),
```

Comment:
I just noticed that this removal is not accompanied by any removal of shell script code that would call those subcommands... Should this not be squashed into an earlier commit, then?

Reply:
Indeed, it should be part of the previous commit.